### DISCUSSION TITLE
Questions regarding Explorer Submission

### DISCUSSION THREAD

**Discussion by soharang** [WalletConnect/walletconnect-monorepo#4385] [Developer Support]
Hello, 

Thanks for providing such a great service for wallets and dapps.

I have a few questions regarding explorer submission:

1. Is it necessary to use the Verify API for wallets in order to pass the submission?
2. Can a wallet pass the submission even if some test cases fail? I noticed that the following page was added last week, so I'm wondering if I can make up for failed cases after passing the submission: https://docs.walletconnect.com/cloud/explorer-submission
3. What should I do if my wallet adds a new supported network after passing the submission? Do I need to submit again, or are there any steps to modify the wallet?

Thanks.

**Comment by glitch-txs** [comment]
Hi, we're working on a new approval process as we want to make sure all approved wallets are working properly. All test cases are mandatories for new wallets.

**Comment by soharang** [comment]
Thanks for the quick confirmation.
